examples/siro_sandbox/sandbox_app.py

```diff
@@ -363,7 +363,10 @@ def _reset_environment(self):
 
         # Reset all the app states
         for app_state in self._app_states:
-            app_state.on_environment_reset(self._episode_recorder_dict)
+            app_state.on_environment_reset(
+                self._episode_recorder_dict,
+                self._episode_helper.current_episode.scene_id,
```

Comment:
This shouldn't be necessary to add this argument. From AppStateFetch, have you tried sandbox_server.episode_helper.current_episode?